models/CsvImporter.php

```diff
@@ -37,10 +47,34 @@
  * @package taoSubjects
 
  */
-class CsvImporter extends \tao_models_classes_import_CsvImporter
+class CsvImporter extends tao_models_classes_import_CsvImporter
 {
+    private function addResourceImportedCallback($form)
```

Comment:
Is it possible to add typehints for `$form` and `return`?

Reply:
Nope, form can be an array of a form object